.gitignore

```diff
@@ -49,6 +49,7 @@ wheels/
 .installed.cfg
 *.egg
 MANIFEST
+great_expectations/
```

Comment:
if we are no longer cloning, do we need this anymore?

Reply:
yes the `great_expectations init` creates this folder